src/services/config/i18next.js

```diff
@@ -23,12 +26,27 @@ i18next
   .use(initReactI18next)
   // for all options read: https://www.i18next.com/overview/configuration-options
   .init({
-    fallbackLng: FALLBACK_LANGUAGE,
```

Comment:
Shouldn't we keep the fallback language option? Removing it might break the webapp for users that don't have either EN or FR as default language.
Also, if we remove this line, then we should remove the whole declaration of `FALLBACK_LANGUAGE` in the services/configuration file